src/lib/Scenes/Map/Bucket.ts

```diff
@@ -38,6 +34,9 @@ export const bucketCityResults = (viewer: GlobalMap_viewer): BucketResults => {
   }
 }
 
+export type BucketKey = keyof ReturnType<typeof bucketCityResults>
+export type BucketResults = ReturnType<typeof bucketCityResults>
+
```

Comment:
`ReturnType` has been my fav for a while, it's what powers the types for resolvers in MP. It can give a name to a inferred set of types. So the keys and the results are now entirely "dynamic" types - change the return value, and these will all change too 🎉